Sources/Clappr/Classes/Extension/Array+appendOrReplace.swift

```diff
@@ -16,3 +16,20 @@ extension Array where Element == Plugin.Type {
         self.append(contentsOf: result)
     }
 }
+
+extension Array where Element == Playback.Type {
```

Comment:
Maybe we could increment `ArrayAppendOrReplaceTests` with this new extension. What do you think?